configs/mount.go

```diff
@@ -18,4 +18,10 @@ type Mount struct {
 
 	// Relabel source if set, "z" indicates shared, "Z" indicates unshared.
 	Relabel string `json:"relabel"`
+
+	// Optional Command to be run before Source is mounted.
+	PremountCmd string `json:"premountcmd"`
```

Comment:
i think that this's better as a list of commands, as user should be able to perform several commands together.